FabulousContacts/FabulousContacts.fsproj

```diff
@@ -25,7 +29,7 @@
     <PackageReference Include="Xam.Plugin.Media" Version="4.0.1.5" />
     <PackageReference Include="Plugin.Permissions" Version="3.0.0.12" />
     <PackageReference Include="Fabulous.XamarinForms.Maps" Version="0.40.3" />
-    <PackageReference Include="FSharp.Core" Version="4.6.2" />
+    <PackageReference Include="FSharp.Core" Version="4.5.2" />
```

Comment:
Usually I do something like this:
```suggestion
    <PackageReference Include="FSharp.Core" />
    <!-- workaround for VSMac bug https://github.com/mono/monodevelop/pull/5137 -->
    <PackageReference Update="FSharp.Core" Version="4.6.2" />
```

Does it fix the detected downgrade?
(Can't test it for now...)

Reply:
The way I did is the way the templates come in VSMac. I tested it, it does work. But if you want, I can use your style, just let me know :)

Reply:
>But if you want, I can use your style, just let me know :)

Yes, please. Thank you :)

Reply:
Done :)